addons/product/views/product_views.xml

```diff
@@ -9,7 +9,7 @@
         <field name="arch" type="xml">
             <form string="Product">
                 <header>
-                    <button string="Print Labels" type="object" name="action_open_label_layout" attrs="{'invisible': [('detailed_type', '==', 'service')]}"/>
+                    <button string="Print Labels" type="object" name="action_open_label_layout" attrs="{'invisible': [('detailed_type', 'not in', ('consu', 'product'))]}"/>
```

Comment:
LGTM. The domain corresponds to what is asked. I would ask the team in charge of the `product` module to be sure :smile: